coordinator/Z-Stack_3.x.0/COMPILE.md

```diff
@@ -4,12 +4,14 @@
 1. Download and install [SIMPLELINK-CC13X2-26X2-SDK_4.30.00.54](http://www.ti.com/tool/download/SIMPLELINK-CC13X2-26X2-SDK).
 1. Download and install [Code Composer Studio 10.1.1.00004](http://www.ti.com/tool/CCSTUDIO).
 
+## Note:
+By default the **launchpad** variant of the CC1352P2_CC2652P (= `znp_CC1352P_2_LAUNCHXL_tirtos_ccs.hex`) is build. To build the **other** variant comment `#define LAUNCHPAD_CONFIG 1` in `preinclude.h`.
+
 ## Compiling
 1. Start Code Composer Studio
 1. Go to *File -> Import -> Code Composer Studio -> CCS Projects -> Select* search-directory: `simplelink_cc13x2_26x2_sdk_4_20_00_35/examples/rtos`. Select `znp_CC26X2R1_LAUNCHXL_tirtos_css`, `znp_CC1352P_2_LAUNCHXL_tirtos_css` and `znp_CC2652RB_LAUNCHXL_tirtos_css`. Press *Finish*.
 1. In Code Composer Studio, expand the 3 projects and for each open `znp.syscfg`, change `Minimal Poll Period (ms)` to `1000`, change it back to `100` immediately and save the file.
 1. Go to your CCS workspace and copy `firmware.patch` to the root.
 1. Open Git Bash, go to your CCS root and apply the patch using `git apply firmware.patch --ignore-space-change`.
-1. Build both projects; right click -> *Build project*.
+1. Build all projects; right click -> *Build project*.
```

Comment:
Maybe it's better to but the comment here? As this is were the building happens.

Reply:
Maybe we should wait until we've fixed the .cproject is different every time you generate it (I tried, every time it get random ids :() as we might need new instructions entirely depending on how we  tackle it?

Reply:
Yes you are right, lets check after instructions are updated.